src/clr-angular/forms/styles/_input-group.clarity.scss

```diff
@@ -30,6 +30,11 @@
       @include clr-no-styles-button();
       @include css-var(color, clr-forms-focused-color, $clr-forms-focused-color, $clr-use-custom-properties);
       padding: 0 $clr-forms-baseline * 1.5;
+
+      clr-icon {
+        @include equilateral($clr_baselineRem_0_75);
+        transform: translate(-1 * $clr_baselineRem_1px, -1 * $clr_baselineRem_1px);
```

Comment:
Same question as above, this looks like a layout change, is it related to the color updates?

Reply:
A size increase to a clean multiple of the icon's base size makes them less blurry. The icons themselves look clearer while only being minimally larger. The transform here and in the datepicker just adjusts for the size increase by visually eliminating the spacer around the edge of the SVG.

It was something that annoyed me so I fixed it. 😊